core/test/gather/driver/network-monitor-test.js

```diff
@@ -381,9 +381,9 @@ describe('NetworkMonitor', () => {
 
     it('should find the 0-quiet periods', () => {
       const records = [
-        record({networkRequestTime: 0, networkEndTime: 1}),
```

Comment:
i updated this file because there's no reason our tests should treat these as seconds. (even if the tests still pass)